solr/contrib/ltr/src/java/org/apache/solr/search/LTRQParserPlugin.java

```diff
@@ -151,7 +159,7 @@ public LTRQParser(String qstr, SolrParams localParams, SolrParams params,
     @Override
     public Query parse() throws SyntaxError {
       // ReRanking Model
-      final String modelName = localParams.get(CommonLTRParams.MODEL);
+      final String modelName = localParams.get(LTRFeatureLoggerTransformerFactory.MODEL);
```

Comment:
Ah, good catch, I relocated to `LTRFeatureLoggerTransformerFactory` since that also uses `MODEL` as the key for getting `ModelQuery` from the request context but really those two are separate 'MODEL' constants really. Will change.